packages/browser-utils/src/networkUtils.ts

```diff
@@ -17,6 +17,7 @@ export function serializeFormData(formData: FormData): string {
 
 /** Get the string representation of a body. */
 export function getBodyString(body: unknown, _logger: Logger = logger): [string | undefined, NetworkMetaWarning?] {
+  // fixme: only add body string if `sendDefaultPii` is enabled?
```

Comment:
For this, we'll just need to add `maxRequestBodySize`

Reply:
I think this is only used in conjunction with Replay and the feature being enabled but I might be missing a case